models.models.js

```diff
@@ -50,5 +50,33 @@ exports.getArticle = (articleId) => {
       const article = result.rows[0];
       return article;
     });
+};
 
+exports.getComments = (articleId) => {
+  if (articleId < 1 || articleId > 99999999) {
```

Comment:
I would make a function to check whether the article exists, rather than handling it this way. There could be a situation where a world popular forum has over a billion articles. 

We can check if the article exists and based on that outcome, decide what we want to return, because it's possible that an article could have no comments.